imports/ui/QuestionSidebar.jsx

```diff
@@ -95,18 +97,19 @@ export class QuestionSidebar extends ControlledForm {
    * Set course id & invoke filter
    * @param {Event} e
    */
-  setCourseId(e) {
+  setCourseId (e) {
     let cId = e.target.value
     if (parseInt(cId) !== -1) {
-      //set the corresponding course tag (user can always remove it)
+      // set the corresponding course tag (user can always remove it)
       let tags = this.state.tags
       Meteor.call('courses.getCourseCodeTag', cId, (error, tag) => {
-        //this does not seem to work, it adds it regardless...
-         tlabels = _(tags).pluck('label')
-         if (tag && !tlabels.includes(tag.label)){
-           tags.push(tag)
-           this.setTags(tags)
-         }
+        if (error) return alertify.error('Error: ' + error.error)
```

Comment:
This seems inconsistent with other cases where the one-line if  statements had {} added to them. I think not adding the {} generally makes it more readable.

Reply:
Sure